raw-span-constants/src/main/proto/org/hypertrace/core/span/constants/v1/envoy_attributes.proto

```diff
@@ -17,15 +17,16 @@ enum Envoy {
   ENVOY_ERROR_REASON = 8 [(string_value) = "error.reason"];
   ENVOY_GRPC_STATUS_CODE = 9 [(string_value) ="grpc.status_code"];
   ENVOY_GRPC_MESSAGE = 10 [(string_value) = "grpc.message"];
-  ENVOY_GUID_X_CLIENT_TRACE_ID = 11 [(string_value) = "guid:x-client-trace-id"];
-  ENVOY_GUID_X_REQUEST_ID = 12 [(string_value) = "guid:x-request-id"];
-  ENVOY_HTTP_PROTOCOL = 13 [(string_value) = "http.protocol"];
-  ENVOY_NODE_ID = 14 [(string_value) = "node_id"];
-  ENVOY_RESPONSE_FLAGS = 15 [(string_value) = "response_flags"];
-  ENVOY_RETRY_COUNT = 16 [(string_value) = "retry.count"];
-  ENVOY_STATUS = 17 [(string_value) = "status"];
-  ENVOY_UPSTREAM_ADDRESS = 18 [(string_value) = "upstream_address"];
-  ENVOY_UPSTREAM_CLUSTER = 19 [(string_value) = "upstream_cluster"];
-  ENVOY_USER_AGENT = 20 [(string_value) = "user_agent"];
-  ENVOY_ZONE = 21 [(string_value) = "zone"];
+  ENVOY_GRPC_AUTHORITY = 11 [(string_value) = "grpc.authority"];
```

Comment:
Why did you change the numbering of these proto fields?

Reply:
I wanted to keep the grpc.* tags together.

Reply:
That would be a breaking change!

https://developers.google.com/protocol-buffers/docs/proto#assigning_field_numbers
> These numbers are used to identify your fields in the message binary format, and should not be changed once your message type is in use

Reply:
Yeah. But I don't think we use proto rawconstant values as such. We use the string values of enums that are defined here. Are we storing the numeric values somewhere?

Reply:
@mohit-a21 We have decided *not* to add any more proto constants. These should be Java string constants so why not start doing that? For a while it might be little inconsistent but we'll have to take that step anyways.